src/RA_Dlg_MemBookmark.cpp

```diff
@@ -514,43 +513,39 @@ void Dlg_MemBookmark::PopulateList()
 
     for (auto it = m_vBookmarks.cbegin(); it != m_vBookmarks.cend(); ++it)
     {
-        LV_ITEM item;
-        ZeroMemory(&item, sizeof(item));
+        LV_ITEM item{};
         item.mask = LVIF_TEXT;
         item.cchTextMax = 256;
         item.iItem = m_nNumOccupiedRows;
-        item.iSubItem = 0;
         item.iItem = ListView_InsertItem(hList, &item);
 
-        ASSERT(item.iItem == m_nNumOccupiedRows);
-
+        Ensures(item.iItem == m_nNumOccupiedRows);
         m_nNumOccupiedRows++;
     }
 
     ListView_EnsureVisible(hList, m_vBookmarks.size() - 1, FALSE); // Scroll to bottom.
-    //ListView_EnsureVisible( hList, topIndex, TRUE ); // return to last position.
+    // Don't sort here
```

Comment:
What I meant is don't sort the vector. We could sort the ListView here though, but don't know how to do it. Would need to research and learn it.